velero/backup/common-service/label-common-service.sh

```diff
@@ -267,7 +267,9 @@ function label_ibm_catalogsources() {
 function label_ns_and_related() {
 
     title "Start to label the namespaces, operatorgroups and secrets... "
-    namespaces=$(${OC} get configmap namespace-scope -n $OPERATOR_NS -oyaml | awk '/^data:/ {flag=1; next} /^  namespaces:/ {print $2; next} flag && /^  [^ ]+: / {flag=0}')
+    # namespaces=$(${OC} get configmap namespace-scope -n $OPERATOR_NS -oyaml | awk '/^data:/ {flag=1; next} /^  namespaces:/ {print $2; next} flag && /^  [^ ]+: / {flag=0}')
+    namespaces=$(${OC} get configmap common-service-maps -n kube-public -o jsonpath='{.data.common-service-maps\.yaml}' | awk -v ns="$OPERATOR_NS" 'BEGIN {found=0}/requested-from-namespace:/ {found=0} $0 ~ ns {found=1} found && /^ *-/ {print $2}' | tr '\n' ',' | sed 's/,$/\n/')
```

Comment:
The only problem with this is would alter the permissions for the label-common-service.sh job that CPD uses since the configmap is in kube public and we wouldn't be able to create permissions there when being used as a job. We ask the user to enter in all applicable namespaces as parameters for the script to run. I think we should just make the list of namespaces based on:
- op ns
- serv ns
- tethered namespace(s)
- cert manager ns (if specified)
- licensing ns (if specified)
- ls reporter ns (if specified)